src/ucp/core/ucp_worker.c

```diff
@@ -488,6 +496,8 @@ ucs_status_t ucp_worker_get_address(ucp_worker_h worker, ucp_address_t **address
             goto err_free;
         }
 
+        VALGRIND_CHECK_MEM_IS_DEFINED(rsc_addr, rsc_addr_length);
+
```

Comment:
@artpol84  `VALGRIND_CHECK_MEM_IS_DEFINED` does not only return the first undefined byte, it also prints an error (same way OpenMPI does it, with its MEMCHECKER calls). So here I want to generate an error as early as possible if the address is undefined, and not wait until, for example, OpenMPI's modex code checks it.